guides/hhvm/00-quickstart/01-quickstart.md

```diff
@@ -9,15 +9,15 @@ The infrastructure you need to run code with HHVM is pretty simple:
 * HHVM itself
 * Some Hack or PHP code
 
-HHVM includes a highly functional and performant webserver called [Proxygen](../deployment/hhvm-servers#proxygen) that is default out of the box, so no installation of a third-party web server like `nginx` is required. However, if you are migrating to HHVM from a PHP runtime engine, and you were already using `nginx` or `apache`, and possibly FastCGI, there are [detailed instructions](../deployment/hhvm-servers#fastcgi) on getting that set up as well. 
+HHVM includes a highly functional and performant web server called [Proxygen](../deployment/hhvm-servers#proxygen) that is availble by default out-of-the-box, so no installation of a third-party web server like `nginx` is required. However, if you are migrating to HHVM from a PHP runtime engine, and you were already using `nginx` or `apache`, perhaps with FastCGI, there are [detailed instructions](../deployment/hhvm-servers#fastcgi) on getting those configurations set up as well.
```

Comment:
Found a plugin for Sublime that helps spell check markdown as I edit. Found it a bit into the writing of the docs, but I think the spelling mistakes weren't great.

And according to https://github.com/hhvm/user-documentation/commit/48693af0bc3d259b90bd547b754b1f52f19c3a00 there were about a dozen or so mistakes that have now been fixed.